### ISSUE TITLE
Import - řazení a zpřehlednění dávek

### ISSUE THREAD

**Issue by luckajirku** [proarc/proarc-client#497]
Dávky v importu se řadí podle názvu a je to docela nepřehledné, když použité jsou mezi novými. U starého aspoň bylo vpravo "načteno", tady není vidět nic, navíc i když zaškrtávací čtvereček zešedne, tak kolečko ke znovunačtení je tak výrazné, že ty dávky naopak poutají větší pozornost než ty nenačtené.
![import](https://github.com/proarc/proarc-client/assets/6359479/1675e43b-ed38-4b4b-9097-b46fdf7fa545)
Pokud se s tím neplánuje nějaká větší změna jako u správy procesů, pomohlo by, kdyby aspoň:
1. načtené dávky odskakovaly dolů až pod ty nenačtené
2. načtené dávky zešedly

**Comment by luckajirku** [comment]
nebo mě ještě napadá další řešení, zaškrtávátko "jen nenačtené", které by umožnilo zobrazit jen ty dávky, které se ještě nenačítaly

**Comment by SykoraLukas** [comment]
Bod 1) Změnit řazení nejde, řadí se vždy všechny dávky abecedně.
Bod 2) Toto lze, je záležitostí klienta, jak to bude prezentovat (klidně i možnost zobrazení slova načtené, jako ve starém klientovi)

Zaškrtávátko by asi šlo, ale narážíme na problém s hierarchií složek a s různými profily (archivní profil ti dovolí načíst jiné věci, než profil pro načtení NDK balíčku a než defaultní profil pro načtení nových dávek) - jak by se to potom mělo chovat, pokud překliknu na jiný profil (vše načítat znovu)?

**Comment by luckajirku** [comment]
a když zmiňuješ hierarchii, nešlo by udělat, že by se ta šipka vepředu ukazovala jen u složek, které mají podložky?

**Comment by SykoraLukas** [comment]
To je otázka na @albertoh. Ale když mu z Api pošlu info, ze složka už nemá podslozky tak by s tím asi mohl pracovat jinak.

**Comment by luckajirku** [comment]
ok, tak uvidíme, co řekne @albertoh . Každopádně když to nejde řadit jinak než abecedně, tak by nám pro zpřehlednění pomohlo to zešednutí načtených, plus přidat sloupeček s Načteno, jak je to v jádře.

**Comment by albertoh** [comment]
Pridal jsem sloupecek a zesednuti.
Az se Lukas vrati domluvim se s nim na dalsi moznosti

**Comment by luckajirku** [comment]
ten sloupeček s "načteno" by byl fajn, pokud by si ho člověk mohl posunout blíž k názvu dávky (ve starém to jde), takhle, jak je úplně vpravo, to moc nepomůže.

**Comment by SmejkalovaAnna** [comment]
Když se mi nepodaří import z jakého koliv důvodu a hodí to chybu, tak se automaticky u dané dávky nezobrazí šipička pro smazání pomocných souborů, musím dát F5 a pak se objeví, ale zase se mi z horního výběru profilu a zařízení vyhodí vybrané nastavení ...
 chybka (ta není důležitá, je jen nasimulovaná):
![obrazek](https://github.com/user-attachments/assets/5daf6cf7-b8e5-43b4-9474-6aed24df23c5)
 odklepnu zavřít a (chybí kulatá šipička, musím dát F5 aby vyskočila): 
![obrazek](https://github.com/user-attachments/assets/48715c61-59b6-418f-97d0-d87734cfd5dd)

**Comment by tumovam** [comment]
Jen doplním za zpracovatele v KNAV: 

- [x] Zešednutý čtvereček po načtení nám přijde v pořádku.
- [x] Výraznější "kolečko" pro znovunačtení se nám též líbí,  aspoň je na první pohled vidět, že jde dávka znovu načíst.
- [x] Šipka "zobáček" > na začátku by se mohl zobrazoval pouze, pokud dávka obsahuje podsložky, ale nevadí nám ani varianta s > a bez podsložek.
- [ ] Zpracovatelům přijde důležité, aby sloupec Načteno byl "blíž", tedy více více vlevo u názvu složky. 

Zbývá ještě něco?

**Comment by SykoraLukas** [comment]
Šipka pro smazání a povolení znovunačtení by měla být defaultně zobrazená všech složek, které mají alespoň u jednoho profilu hodnotu načteno. Bez ohledu na to, jaký mám vybraný profil nahoře v šedé liště.

**Comment by SykoraLukas** [comment]
Zobrazení `Načteno`. Jsou zatím 2 možnosti, jak to zobrazit. Buďto hned za název každé složky to dát do závorky nebo ještě před strom jako první sloupec a teprve do druhého sloupce dát ten seznam (a jejich hierarchii) složek k načtení.

Možnost druhá je znázorněná zde:

![Image](https://github.com/user-attachments/assets/a83b4758-70c1-49ce-a05c-f4ecae7e8e0e)

Vyhovoval by nějaký návrh všem?

**Comment by SmejkalovaAnna** [comment]
@SykoraLukas Mě se líbí tahle druhá zobrazená varianta.....

**Comment by tumovam** [comment]
@luckajirku můžu poprosit o názor zobrazení "Načteno"? Díky

**Comment by albertoh** [comment]
> ještě se pro jistotu zeptám, šířka sloupce nebude napevno, půjde zúžit, OK?

Jak by to vypadalo, kdyby to se zuzilo moc, ze by slovo "Načteno" se neveslo? V tom sloupci bude jen to slovo nebo nic.
Nasadil jsem na testu, abyste se podivali jak to vypada. Bez moznosti zuzeni.
Napiste nazor :)

**Comment by luckajirku** [comment]
Jo, právě proto, že je to buď to slovo, nebo nic, tak stačí, když z něj uvidím kousek:-). Každopádně by i ten nápis načteno měl být šedě, když je takhle vepředu. Cílem mělo být, aby se ty načtené věci upozadily, a pozornost poutaly ty věci k načtení.

**Comment by albertoh** [comment]
Zesedlo.
Udelat sloupec zuzitelny mi da vic prace: soucasny disajn neni na to vhodny

**Comment by nezbedova** [comment]
A co kdyby tam nebylo slovo načteno, ale jen _OK_ ? Mohl by pak být sloupec napevno, ale zúžený na 2 písmena ?

Nebo jiné krátké slovo nebo znak nebo zkratka. Ale momentálně mě nic nenapadá.

**Comment by luckajirku** [comment]
takhle s tím zašedlým nápisem už to je dobrý, za nás to tak může být

čt 20. 2. 2025 v 9:53 odesílatel Alberto Hernandez ***@***.***>
napsal:

**Comment by nezbedova** [comment]
Nám to takto také vyhovuje.

**Comment by albertoh** [comment]
> * [x]  Šipka pro smazání a povolení znovunačtení by měla být defaultně zobrazená všech složek, které mají alespoň u jednoho profilu hodnotu načteno. Bez ohledu na to, jaký mám vybraný profil nahoře v šedé liště.

Hotovo
